docs/reference/security/overview.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[[elasticsearch-security]]
+== Security overview
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+Security protects {es} clusters by:
+
+* <<preventing-unauthorized-access, Preventing unauthorized access>>
+  with password protection, role-based access control, and IP filtering.
+* <<preserving-data-integrity, Preserving the integrity of your data>>
+  with message authentication and SSL/TLS encryption.
```

Comment:
```suggestion
  with SSL/TLS encryption.
```
"message authentication" referred to the "system key" feature that we removed in favour of SSL in 6.0. I hadn't noticed that it was still on the docs until now.